docs/en/observability/application-logs.asciidoc

```diff
@@ -0,0 +1,219 @@
+[[application-logs]]
+= Application logs
+
+Application logs provide valuable insight into events that have occurred within your services and applications.
+Correlating these logs with application traces enables you to view the context in which a log event has occurred.
+
+Structured vs unstructured logs:
+
+Structured logging is where the application writes JSON objects directly--instead of
+parsing your logs into a structured object at ingest time.
+
+Why is structure so important in logs? So you can search and aggregate on individual fields within your logs.
+
+
+[float]
+[[ingest-application-logs]]
+== Ingest application logs
+
+There are four ways to ship your application logs to the {stack}.
+Your use case will determine the method that's right for you.
+
+[float]
+=== Plaintext logs with Filebeat
+
+**Requirements**
+
+* The Elastic APM agent for your programming language.
+* Raw, plain-text application logs
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, filter and correlate multi-line logs, and forwards them to {es}
+
+**Pros**
+
+* All programming languages/frameworks are supported
+
+**Cons**
+
+* Must parse application logs to be useful—meaning writing and maintaining Grok patterns and spending CPU cycles on parsing
+
+Learn more: <<plaintext-with-filebeat>>
+
+[float]
+=== ECS loggers with Filebeat
+
+Elastic Common Schema (ECS) loggers format your logs into ECS-compatible JSON,
+which removes the need to manually parse logs.
+
+**Requirements**
+
+* The Elastic APM agent for your programming language
+* The Elastic ECS logger for your language or framework
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, and forwards them to {es}
+
+**Pros**
+
+* Popular logging frameworks supported
+* Simplicity: no manual parsing
+* Decently human-readable JSON structure
+* APM Log correlation
+* Resilient in case of outages
+
+**Cons**
+
+* Not all frameworks are supported
+
+Learn more: <<ecs-with-filebeat>>
+
+[float]
+=== APM agent ECS reformatting with Filebeat
```

Comment:
```suggestion
=== ECS reformatting with Filebeat
```

We can omit the APM for a shorter title as agent as it's just an implementation detail.